x-pack/test/api_integration/services/index.ts

```diff
@@ -14,6 +14,7 @@ import { services as commonServices } from '../../common/services';
 // @ts-ignore not ts yet
 import { EsSupertestWithoutAuthProvider } from './es_supertest_without_auth';
 import { SecuritySolutionApiProvider } from './security_solution_api.gen';
+import { SecuritySolutionApiProvider as SecuritySolutionExceptionsApiProvider } from './security_solution_exceptions_api.gen';
```

Comment:
we are about to deprecate `x-pack/test` and I already moved most of the services out of `x-pack/test/api_integration/services` path.

Maybe you can create a follow-up PR and move all the Security Solution FTR services and utils (`x-pack/test/common/utils/security_solution`) directly into `x-pack/test/security_solution_api_integration` dir, this way moving the whole directory to `x-pack/solutions/security/test` will be less challenging with imports update?